From 8716901d2215a314504b7df6282aedfcf89da1ea Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
Date: Wed, 22 Aug 2012 01:48:17 +0200
Subject: [PATCH 57/98] dtc/libfdt: install missing header

Previously, only two headers were installed: libfdt.h and fdt.h.
But libfdt.h also #includes libfdt_env.h, which was not installed.

Install this missing header too.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
Acked-by: David Gibson <david@gibson.dropbear.id.au>
---
 libfdt/Makefile.libfdt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libfdt/Makefile.libfdt b/libfdt/Makefile.libfdt
index 4366627..91126c0 100644
--- a/libfdt/Makefile.libfdt
+++ b/libfdt/Makefile.libfdt
@@ -4,7 +4,7 @@
 # be easily embeddable into other systems of Makefiles.
 #
 LIBFDT_soname = libfdt.$(SHAREDLIB_EXT).1
-LIBFDT_INCLUDES = fdt.h libfdt.h
+LIBFDT_INCLUDES = fdt.h libfdt.h libfdt_env.h
 LIBFDT_VERSION = version.lds
 LIBFDT_SRCS = fdt.c fdt_ro.c fdt_wip.c fdt_sw.c fdt_rw.c fdt_strerror.c fdt_empty_tree.c
 LIBFDT_OBJS = $(LIBFDT_SRCS:%.c=%.o)
-- 
1.8.1.2

